### DISCUSSION TITLE
Delete from storage?

### DISCUSSION THREAD

**Discussion by DevOriginalWang** [microsoft/dicom-server#2729] [Q&A] [answered]
When I send a Delete request, it supposedly deletes the study from the study list, but I'm not sure if it actually deletes off the blob storage container in the back. Can anyone confirm this?

**Comment by DevOriginalWang** [comment]
I check in the blob storage and it looks like all the dicom files are still there.

**Comment by jnlycklama** [comment, marked as answer]
Hi,
The DELETE request will delete the DICOM file from blob storage asynchronously, after the DELETE request has been completed. There is a configured DeleteDelay which specifies how long we will wait before attempting to delete the DICOM file - https://github.com/microsoft/dicom-server/blob/64b4322dea2a83d224722bd52f5aec60ef81b01a/src/Microsoft.Health.Dicom.Web/appsettings.json#L83
